### ISSUE TITLE
Casts should be placed on same line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#530]
We should report a violation in case a cast is placed on a different line.

For example, following code should report a violation:
```c#
var xyz =
          (int)
             abc;
```

while following code should *NOT* report a violation:
```c#
var xyz = (int)abc;
```
